### ISSUE TITLE
Broken old v201609 (and possibly other) contracts

### ISSUE THREAD

**Issue by franckyj** [googleads/googleads-dotnet-lib#106]
We're still using the v201609 version of the C# client and previously (in version 18.33.0) and I've noticed that our projects didn't compile anymore.

Looking deeper into the issue I've found that the `mutate` methods of different services have been changed from this
```
public interface ICampaignCriterionService
{
    // ...
    CampaignCriterionReturnValue mutate(CampaignCriterionOperation[] operations);
    // ...
}
```
to this
```
public interface ICampaignCriterionService : CampaignCriterionServiceInterface { }

public interface CampaignCriterionServiceInterface
{
    // ...
    Wrappers.CampaignCriterionService.mutateResponse mutate(Wrappers.CampaignCriterionService.mutateRequest request);
    // ...
}
```
Notice that the parameter changed from being an array of operations to a `mutateRequest` object.

Is there any specific reason why you have changed those methods signatures for a version that is already released?

I've looked at those files
https://raw.githubusercontent.com/googleads/googleads-dotnet-lib/v18.33.0/src/AdWords/v201609/AdWordsApi.cs
https://raw.githubusercontent.com/googleads/googleads-dotnet-lib/master/src/AdWords/v201609/AdWordsApi.cs

Thanks,
Francois

**Comment by franckyj** [comment]
We want to migrate to the newer v201708 version but for a time we test that the output of one matches the output of the other. This is why we have both versions at the same time.

**Comment by christopherseeley** [comment]
We changed the SOAP toolkit in v22.0 and this looks like a bug in our code generation. 
The concrete class CampaignCriterionService still has the old method signature - the interface just exposes the SOAP wrapper class.

**Comment by christopherseeley** [comment]
This will be fixed in the next release.

**Comment by christopherseeley** [comment]
Fixed in v21.1.1
